src/commands/init.ts

```diff
@@ -0,0 +1,80 @@
+import { Input } from "@oclif/parser";
+import * as fs from "fs-extra";
+import * as path from "path";
+import cli from "cli-ux";
+
+import TartCommand from "../TartCommand";
+
+export default class Init extends TartCommand {
+  static description = "Initializes tart for this project";
```

Comment:
maybe put examples?

Reply:
I added some examples but I don't know if they're all that great